### PR TITLE
Topic: Sleep

### PR DESCRIPTION
**Pull request by jamzi**
Link to the article: https://deploy-preview-7--devhealthio.netlify.com/sleep/

If there are some paragraphs that are unclear, tell me and I'll add more context.

### CODE REVIEW

File: content/blog/sleep/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: Sleep
+date: "2019-05-08T17:12:03.284Z"
+description: I'll sleep when I'm dead.
+---
+
+> A condition of body and mind which typically recurs for several hours every night, in which the nervous system is inactive, the eyes closed, the postural muscles relaxed, and consciousness practically suspended (Oxford dictionary).
+
+Studies have shown that 1 out of every 2 adults in America is not getting the recommended hours of sleep. 1 out of 3 people is trying to survive on 6 hours or less.
+
+Adults need from 7 to 9 hours of sleep per day and the average American is sleeping **6.5 hours** during the week. This used to be **7.9 hours** in 1942.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _per day_

a comma after 'day'

File: content/blog/sleep/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+There are numbers of important processes happening during the night like muscle repair and grow as well as a body cleanse. There is a sewage system in the brain cleaning the brain of metabolic toxins that we accumulated during the day.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _grow_

'growth' if you mean a noun describing the process of growing

Comment (tinapetan) [inline content/blog/sleep/index.md]: _accumulated_

accumulate

File: content/blog/sleep/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+> Like humans, most animals sleep for at least a few hours. Dolphins are interesting because they are conscious breathers and in order to not drown, they need to go to the surface and breathe. They float and shut half of its brain during sleep and keep the other half active to keep the basic body functions active.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _... a few hours._

add 'a day', 'a night', 'per day', 'per night' or sth like that after 'hours'

Comment (tinapetan) [inline content/blog/sleep/index.md]: _Dolphins are interesting ..._

it's quite a long and complicated sentence, i would rephrase it a bit. sth like 'Dolphins are interesting./Dolphins are an interesting example. They are conscious breathers...'

Comment (tinapetan) [inline content/blog/sleep/index.md]: _in order to not drown_

not wrong but 'in order not to drown' is grammatically more acceptable

Comment (tinapetan) [inline content/blog/sleep/index.md]: _its brain_

probably 'their brain'

Comment (tinapetan) [inline content/blog/sleep/index.md]: _to keep the basic ..._

'maintain', 'preserve' or any other synonym instead of 'keep' since 'keep' is already used a few words before this one

File: content/blog/sleep/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+`youtube: https://www.youtube.com/watch?v=1ZSzI5mHwS8`
+
+Let's explore what affects sleep.
+
+### What is a circadian rhythm?
+
+It's a 24-hour internal clock, a regulator for sleepiness and wakefulness. When we feel the most energized can vary a bit from person to person, but for most adults, there is a noticeable drop in energy from 1 to 3 in the afternoon and from 2 to 4 at night.
+
+Outside factors like lightness and darkness can also have an impact on the internal clock. The brain sends a signal to the body to release melatonin which makes you sleepy.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _Outside factors like lightness and darkness_

Outside factors, such as lightness and darkness, ...

File: content/blog/sleep/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### Effects of sleep deprivation
+
+Human beings are the only species that deprive themselves of sleep and we are not wired to compensate for the lack of sleep. Sleep is not like a bank. We can't accumulate a debt and hope to pay it off on the weekend.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _and we are not wired_

leave 'we' out

File: content/blog/sleep/index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+![Airport sleep](./airport-sleep.jpg)
+
+Effects of sleep deprivation can show:
+
+- **Worsened brain performance**. Getting less than 6 hours of sleep will drop the time to physical exhaustion by 30%.
+
+- **Lower muscle strength**. The less sleep you have less muscle strength, as well as jump height and running speed decrease.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _The less sleep you have less muscle strength, as well as jump height and running speed decrease._

The less sleep you have, the lower your muscle strength, jump height and running speed.

or

The less sleep you have, the lower your muscle strength. Your jump height and running speed may decrease as well.

File: content/blog/sleep/index.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Lack of testesterone**. Men who sleep only 5 to 6 hours a night will have a level of testosterone 6-10 years their senior.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _Lack of testesterone_

a typo. 'testosterone' is correct

File: content/blog/sleep/index.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+Long term lack of sleep is also connected to diseases, one of them is Alzheimer's disease. This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _Long term lack of sleep is also connected to diseases, one of them is Alzheimer's disease._ 

I would put that as the fourth bullet point, just like 'worsened brain performance', etc. (or at least a separate paragraph from the following sentence; they don't work well together)

you can write sth like '**Diseases**. Long term lack of sleep is also connected to diseases, such as Alzheimer's.'

you can leave out 'disease' after 'Alzheimer's' so that you don't repeat it for the third time; the meaning is the same: https://en.wikipedia.org/wiki/Alzheimer%27s_disease

Comment (tinapetan) [inline content/blog/sleep/index.md]: _This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined._

this sentence can either be the fifth bullet point or just added bellow as it is now. just replace 'and' with 'or' at the end, because the combination of alcohol and drugs takes both, not just one of the two.

File: content/blog/sleep/index.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+### How to improve sleep?
+
+Here are some tips to help you with night's sleep:
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _night's sleep_

maybe 'a good night's sleep'. but it's okay either way

File: content/blog/sleep/index.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Keep a regular sleep schedule**. Go to bed at the same time every day, try to do it for weekends too. This way, you'll train the body to fall asleep at the same time.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _try to do it for weekends too_

i would say 'even on weekends if possible'

Comment (tinapetan) [inline content/blog/sleep/index.md]: _... the body to fall asleep at the same time_

maybe 'the body to always fall asleep at the same time'

or

'the body to fall asleep at the same time every day/night'

File: content/blog/sleep/index.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Avoid screen time before bed**. Blue light, emitted from mobile phones can postpone melatonin production, which is needed for the body to feel sleepy. One hour of screen time can postpone sleepiness for up to 3 hours.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _mobile phones_

a comma after 'phones'

File: content/blog/sleep/index.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Cool, clean and quiet bedroom**. Bedroom temperature should be between 60 and 67 degrees (15 to 20 degrees Celsius). Eliminate any noises that can disturb sleep and blackout any sources of light from outside lamps or electronic devices.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _between 60 and 67 degrees_ 

you should define the scale (probably Fahrenheit) as you did for the Celsius.

Comment (tinapetan) [inline content/blog/sleep/index.md]: _blackout_

black out

File: content/blog/sleep/index.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Eat 2 to 3 hours before bed**. This way, you won't be too hungry or too full going to bed.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _you won't be too hungry or too full going to bed._

either

'you won't be too hungry or too full when going to bed'

or

'you won't go to bed too hungry or too full'

File: content/blog/sleep/index.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+![Bedroom](./bedroom.jpg)
+
+### "There is an app for that."
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _"There is an app for that."_

if you mean that as a joke (https://www.urbandictionary.com/define.php?term=There%27s%20an%20app%20for%20that), then it's okay. if not, leave out the quotation marks.

File: content/blog/sleep/index.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+To measure the quality of your sleep, I recommend downloading an app called [Sleep as Android](https://play.google.com/store/apps/details?id=com.urbandroid.sleep), available for Android.
+
+It allows you to track sleep and use an ability to set up "smart" alarms, waking you up at the end of the sleep cycle. There are similar apps for iOS as well.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _use an ability_

i'm not sure if you can say that sth/sb ''uses'' the ability. 'has' would be better here instead of 'use' (and yes, put it in the singular as it probably doesn't refer back to 'allows' but to 'it')

also, 'the' instead of 'an'

Comment (tinapetan) [inline content/blog/sleep/index.md]: _There are similar apps for iOS as well._

if you know of any good ones for iOS, write them up. if not, leave it like that

File: content/blog/sleep/index.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+![Sleep as Android 1](./sleep-as-android.png)
+
+### Conclusion
+
+Sleep is really important for optimal health and well-being. If you would like to get more sleep, try some of the tips above.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _really important_

you can say 'significant' or sth like that

File: content/blog/sleep/index.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+**Remember, less sleep doesn't mean more productivity**.
```
Comment (tinapetan) [inline content/blog/sleep/index.md]: _Remember, ..._

this sounds like an addition, another piece of advice. i would say ‘And remember, ...’

File: content/blog/sleep/index.md (hunk 14 above)
Comment (jamzi) [reply content/blog/sleep/index.md]: Yes, it's a saying / phrase.

File: content/blog/sleep/index.md (hunk 17 above)
Comment (jamzi) [reply content/blog/sleep/index.md]: Good call.
